src/app/frontend/deploy/namespace_dialog.html

```diff
@@ -0,0 +1,33 @@
+<!--
```

Comment:
Re naming of the files. The general naming idea is to to have names with no separators and then suffix it with functional or angular suffixes. So we have `_test.js`, `_po.js` or `_controller.js`. How about naming this thing a "create namespace" and have files: `createnamespace.js` `createnamespace_controller.js` `createnamespace.html` and `createnamespace_dialog.js`. This is promoting `_dialog.js` to a set of suffixes we can use for angular constructs.

What do you think?